### DISCUSSION TITLE
How to install workerd in Arm64?

### DISCUSSION THREAD

**Discussion by byteshiva** [cloudflare/workerd#44] [General]
Are there any perquisite steps required to install workerd in Arm64 machines.

**Comment by kentonv** [comment]
It should be the same as x64. We currently publish binaries on npm for Linux-x64, Linux-arm64, MacOS-x64, and MacOS-M1.

**Reply by byteshiva** [reply]
I see its published on npm,

https://www.npmjs.com/package/workerd
